holidays/calendars/hindu.py

```diff
@@ -10,220 +10,77 @@
 #  Website: https://github.com/vacanza/holidays
 #  License: MIT (see LICENSE file)
 
+from collections.abc import Iterable
 from datetime import date
 from typing import Optional
 
 from holidays.calendars.custom import _CustomCalendar
-from holidays.calendars.gregorian import JAN, FEB, MAR, OCT, NOV
+from holidays.calendars.gregorian import JAN, FEB, MAR, APR, OCT, NOV, AUG, SEP, DEC
+from holidays.helpers import _normalize_tuple
 
 DIWALI = "DIWALI"
```

Comment:
To avoid problems with other countries, I suggest add a separate holiday like
`DIWALI_INDIA = "DIWALI_INDIA"`